Alliance.Client/Extensions/ExNativeUI/AgentStatus/ViewModels/AgentInteractionInterfaceVMCustom.cs

```diff
@@ -0,0 +1,616 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.Core;
+using TaleWorlds.Engine;
+using TaleWorlds.InputSystem;
+using TaleWorlds.Library;
+using TaleWorlds.Localization;
+using TaleWorlds.MountAndBlade.MissionRepresentatives;
+using TaleWorlds.MountAndBlade;
+using Alliance.Common.Core.ExtendedXML.Models;
+using Alliance.Common.Core.ExtendedXML.Extension;
+using System.Data.SqlTypes;
+using TaleWorlds.CampaignSystem.ViewModelCollection.CharacterDeveloper;
+using TaleWorlds.LinQuick;
+using TaleWorlds.MountAndBlade.Network.Gameplay.Perks.Effects;
+using static TaleWorlds.MountAndBlade.UsableMissionObject;
+using Alliance.Common.Core.Configuration.Models;
+
+
+namespace Alliance.Client.Extensions.ExNativeUI.AgentStatus.ViewModels
+{
+	public class AgentInteractionInterfaceVMCustom : ViewModel
```

Comment:
Also, I can see `AgentInteractionInterfaceVMCustom` and `MissionGauntletMainAgentEquipmentControllerViewCustom` have a lot in common. Shouldn't we create a common abstract parent class ?

Reply:
They already have ViewModel as a abstract parent class